src/PrestaShopBundle/Resources/views/Admin/Layout/default_layout.html.twig

```diff
@@ -112,9 +112,7 @@
              title="{{ maintenanceTitle|e('html') }}"
              href="{{ url('admin_maintenance') }}"
           >
-            <i class="material-icons"
-               style="{% if ps.frontOfficeAccessibleForAdmins|default(false) %}color: var(--green);{% endif %}"
```

Comment:
@tblivet functional change

Reply:
Indeed the change of color should still be present, it can be improved to be based on a class and I guess the color can be adapted to match the new design but it shouldn't be removed completely, we need a color based differenciation